online-fishing-store/src/app/pages/profile-page/personal-data/personal-data.component.html

```diff
@@ -0,0 +1,118 @@
+
+<div class="personalData">
+    <div class="fixed-avatar">
+        <img
+            class="button__img "
+            [src]="profilePicture"
+            [alt]="avatarImgAltAttributeValue"
+        />
+    </div>
+    <form class="form" [formGroup]="form">
+        <div class="avatar form-control">
+            <div class="container">
+                <input 
+                    type="file" 
+                    accept="image/*,image/jpeg"
+                    name="avatar"
+                    [attr.disabled]="isEditAvatar ? 'disabled' : null"
+                    formControlName="avatar">
+                <div  class="button-enable" *ngIf="isEditAvatar"(click)="isEditAvatar = !isEditAvatar"></div>
+                <div  class="button-disable" *ngIf="!isEditAvatar"(click)="isEditAvatar = !isEditAvatar"></div>                
+            </div>
+        </div>
+        <div class="firstName form-control">
+            <div class="container" >
+                <h2 class="value" >{{userData.firstName}}</h2>
+                <div class="button-enable" *ngIf="isEditFirstName"(click)="isEditFirstName = !isEditFirstName"></div>
+                <div class="button-disable" *ngIf="!isEditFirstName"(click)="isEditFirstName = !isEditFirstName"></div>
+            </div>
+            <input 
+            type="text" 
+            [(ngModel)]="userData.firstName" 
```

Comment:
You can't use ngModel together with reactive forms. Open documentation and rework it